src/Deprecations.jl

```diff
@@ -1,3 +1,4 @@
 # Deprecated in 0.10.*
 
 @deprecate automorphisms(x::Graph) automorphism_group_generators(x)
+@deprecate _containement_helper(R::MPolyRing, N::Int, M::Int, I::MPolyIdeal, W::Vector, ord::Symbol) _containment_helper(R, N, M, I, W, ord)
```

Comment:
This is an internal only function, isn't it? So I don't think we need to deprecate it; or do you have reason to believe there's external code using it?

Reply:
I just thought it might be best practice. But probably it is overly careful. I remove the `@deprecate`.